main.c

```diff
@@ -1067,6 +1073,13 @@ int main(int argc, char *argv[]) {
 		display_event, NULL);
 	wl_event_source_check(source);
 
+	if (daemonize) {
+		if (daemon(1, 1) != 0 ) {
```

Comment:
This function sounds non-standard (not in POSIX). Here's a standard one: https://github.com/swaywm/swaylock/blob/c46a5197a24c7597404199165763239eb60fe714/main.c#L60

Reply:
Hm, not sure why a pipe is involved there…

Reply:
not an ideal one also, it doesn't check if fork failed.